canvas_modules/common-canvas/src/common-properties/properties-main/properties-main.jsx

```diff
@@ -103,6 +103,9 @@ class PropertiesMain extends React.Component {
 
 	UNSAFE_componentWillReceiveProps(newProps) { // eslint-disable-line camelcase, react/sort-comp
 		if (newProps.propertiesInfo) {
+			if (this.props.light !== newProps.light) { // set the new light prop in controller
```

Comment:
Is there a reason this is under `if (newProps.propertiesInfo) {`?  I would think this check can be outside of that.

Reply:
@matthoward366 Tested by keeping it outside the propertiesInfo check and working fine. Code updated.